templates/controllers/tab/settings/navigationMenus/form/navigationMenuSettingsForm.tpl

```diff
@@ -0,0 +1,32 @@
+{**
+ * controllers/tab/settings/navigationMenus/form/NavigationMenuSettingsForm.tpl
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2003-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * NavigationMenus settings form.
+ *
+ *}
+
+{* Help Link *}
+{help file="settings.md" section="website" class="pkp_help_tab"}
+
+<script type="text/javascript">
+	$(function() {ldelim}
+		// Attach the form handler.
+	    $('#navigationMenuSettingsForm').pkpHandler('$.pkp.controllers.form.AjaxFormHandler');
+	{rdelim});
+</script>
+
+<form id="navigationMenuSettingsForm" class="pkp_form" method="post" action="{url router=$smarty.const.ROUTE_COMPONENT component="tab.settings.WebsiteSettingsTabHandler" op="saveFormData" tab="navigationMenus"}">
+	{csrf}
```

Comment:
Since it's not a form, you can remove the `<script>`, `<form>` and `{csrf}` bits.